### ISSUE TITLE
Questions about SHHQ-1.0 FID

### ISSUE THREAD

**Issue by SeanChen0220** [stylegan-human/StyleGAN-Human#27]
Hi!
Your work is so meaningful!
Nowadays I'm trying to caculate the FID score using your released model trained with SHHQ-1.0 (~40K, SHHQ-1.0_sg2_512.pk), then I use it to generate 50K images and caculate FID with SHHQ-1.0, but my FID score (7.12) is much higher than that you reported (3.68).
Could you share more details about your reported FID? What images you use to caculate FID? Is there a test set using for FID?
By the way, could you release your training settings of 40K model and 230K model, it is not clear that if they are trained with similar iters or epochs.

**Comment by stylegan-human** [comment]
Hi, we just released the training code for sg2 and sg3. 
1. We trained SHHQ-1.0 several times with sg2 and always got fid around 3.7. The main modification of sg2 is using [8 layers for mapping instead of 2](https://github.com/stylegan-human/StyleGAN-Human/blob/main/training_scripts/sg2/train.py#L169). 
2. As for FID, we use the default version in the original code of stylegan1-3.
3. We basically chose the model with the lowest FID. And for different dataset sizes, of course larger dataset takes longer to converge. In our case, the best model for SHHQ-1.0(40K) with FID 3.68 takes 12000kimgs, and the best model trained with our full dataset (230K) takes 35000kimgs to get FID 1.57.

**Comment by SeanChen0220** [comment]
Thank you very much! May I ask about the training batch size for SHHQ-1.0(40K)?

**Comment by stylegan-human** [comment]
> 

our batch-gpu is 4.

**Comment by koutilya-pnvr** [comment]
@SeanChen0220 can you share how you ended up getting 7.12 FID in your earlier experiments! Did you use the SHHQ-1.0 40K images as reference to compute the FID for the generated images? More details please.

**Comment by SeanChen0220** [comment]
@koutilya-pnvr Yeah, SHHQ1.0-40K was used to compute FID-50K in my earlier exp, and I have found the problem. When I disabled truncation when generating samples for fid calculation, I got a similar FID to that in the paper (around 3.7).
